gce/prepull.yaml

```diff
@@ -14,122 +14,83 @@ spec:
+        name: agnhostprivate
         command: ["cmd", "exit"]
-      - image: e2eteam/test-webserver:1.0
-        name: test-webserver4
+      - image: e2eteam/apparmor-loader:1.0
+        name: apparmor-loader
         command: ["cmd", "exit"]
-      - image: e2eteam/cassandra:v13
-        name: cassandra5
+      - image: e2eteam/busybox:1.29
+        name: busybox1
         command: ["cmd", "exit"]
       - image: e2eteam/dnsutils:1.1
         name: dnsutils6
         command: ["cmd", "exit"]
       - image: e2eteam/echoserver:2.2
         name: echoserver7
         command: ["cmd", "exit"]
-      - image: e2eteam/entrypoint-tester:1.0
-        name: entrypoint-tester8
-        command: ["cmd", "exit"]
-      - image: e2eteam/etcd:v3.3.10
+      - image: e2eteam/etcd:v3.4.3
         name: etcd9
         command: ["cmd", "exit"]
-      - image: e2eteam/etcd:3.3.10
-        name: etcd10
-        command: ["cmd", "exit"]
-      - image: e2eteam/fakegitserver:1.0
-        name: fakegitserver11
-        command: ["cmd", "exit"]
       - image: e2eteam/gb-frontend:v6
         name: gb-frontend12
         command: ["cmd", "exit"]
       - image: e2eteam/gb-redisslave:v3
         name: gb-redisslave13
         command: ["cmd", "exit"]
-      - image: e2eteam/hazelcast-kubernetes:3.8_1
-        name: hazelcast-kubernetes14
+      - image: e2eteam/httpd:2.4.38-alpine
+        name: httpd2-4-38
         command: ["cmd", "exit"]
-      - image: e2eteam/hostexec:1.1
-        name: hostexec15
+      - image: e2eteam/httpd:2.4.39-alpine
+        name: httpd2-4-39
         command: ["cmd", "exit"]
-      - image: e2eteam/iperf:1.0
-        name: iperf16
+      - image: e2eteam/ipc-utils:1.0
+        name: ipcutils
         command: ["cmd", "exit"]
       - image: e2eteam/jessie-dnsutils:1.0
         name: jessie-dnsutils17
         command: ["cmd", "exit"]
       - image: e2eteam/kitten:1.0
         name: kitten18
         command: ["cmd", "exit"]
-      - image: e2eteam/liveness:1.1
-        name: liveness19
-        command: ["cmd", "exit"]
-      - image: e2eteam/logs-generator:1.0
-        name: logs-generator20
-        command: ["cmd", "exit"]
       - image: e2eteam/mounttest:1.0
         name: mounttest21
         command: ["cmd", "exit"]
       - image: e2eteam/nautilus:1.0
         name: nautilus22
         command: ["cmd", "exit"]
-      - image: e2eteam/net:1.0
-        name: net23
-        command: ["cmd", "exit"]
-      - image: e2eteam/netexec:1.1
-        name: netexec24
-        command: ["cmd", "exit"]
-      - image: e2eteam/nettest:1.0
-        name: nettest25
-        command: ["cmd", "exit"]
       - image: e2eteam/nginx:1.14-alpine
         name: nginx26
         command: ["cmd", "exit"]
       - image: e2eteam/nginx:1.15-alpine
         name: nginx27
         command: ["cmd", "exit"]
-      - image: e2eteam/no-snat-test:1.0
-        name: no-snat-test28
+      - image: e2eteam/nonewprivs:1.0
+        name: nonewprivs
```

Comment:
Image is not for Windows. The image is supposed to print the EUID, which is not Windows-specific.

Reply:
Thanks, removed.